.pre-commit-config.yaml

```diff
@@ -94,7 +94,7 @@ repos:
   #     - id: absolufy-imports
   #       files: ^backend/
   - repo: https://github.com/asottile/pyupgrade
-    rev: v3.15.1
+    rev: v3.15.2
```

Comment:
@hari-kuriakose we have 2 pyupgrade hooks running with 
`pyupgrade --py38-plus --keep-runtime-typing` and `pyupgrade --py39-plus --keep-runtime-typing`. Which do you think can be removed here?

Reply:
@chandrasekharan-zipstack We need only one. Let's remove the other `pyupgrade --py38-plus --keep-runtime-typing` hook.

Reply:
Handled in #245